lib/iris/tests/experimental/test_animate.py

```diff
@@ -40,6 +40,7 @@
 @tests.skip_plot
 class IntegrationTest(tests.GraphicsTest):
     def setUp(self):
+        super(IntegrationTest, self).setUp()
```

Comment:
We need to make sure that the `GraphicsTest.setUp` is called, as it `aquires` the non re-entrant lock that protects plot figures for a test.